### ISSUE TITLE
In Value Stream Map clicking "more..." link no longer works

### ISSUE THREAD

**Issue by chrislovsund** [gocd/gocd#2776]
##### Summary

Viewing the VSM and pressing "more..." no longer works.  

###### Basic environment details

* Go Version: `16.10.0 (4131-730ff1867576754414cc632957f344d0263bc06d)`
* JAVA Version: `1.7.0_111`
* OS: `Linux 3.10.0-327.36.1.el7.x86_64`
* Browser vendor and version (if relevant): `chrome, firefox, safari`

###### Additional Environment Details

```plain
{
  "Timestamp": "2016-10-10T13:25:13+02:00",
  "Config Statistics": {
    "Valid Config": {
      "Number of pipelines": 211,
      "Number of agents": 45,
      "Number of environments": 15,
      "Number of unique materials": 178,
      "Number of schedulable materials": 171
    },
    "Security": {
      "LDAP": true,
      "Password": false
    }
  },
  "Config file locations": {
    "loc.config.dir": "/etc/go",
    "loc.log.root.0": "/var/log/go-server",
    "loc.log.basename.0": "go-server.log",
    "loc.log.root.1": "/var/log/go-server",
    "loc.log.basename.1": "go-shine.log"
  },
  "Go Server Information": {
    "Version": "16.10.0 (4131-730ff1867576754414cc632957f344d0263bc06d)"
  },
  "Server Health Information": {
    "Messages Count": 0
    ]
  },
  "Cache Information": {
    "Cache configuration information": {
      "name": "cruise",
      "Maximum Elements in Memory": 100000,
      "Maximum Elements on Disk": 0,
      "Memory Store Eviction Policy": {
        "myName": "LRU"
      },
      "Clean or Flush": true,
      "Eternal": true,
      "Time To Idle Seconds": 120,
      "time To Live Seconds": 120,
      "Overflow To Disk": false,
      "Disk Persistent": false,
      "Disk Store Path": null,
      "Disk Spool Buffer Size in MB": 30,
      "Disk Access Stripes": 1,
      "Disk Expiry Thread Interval Seconds": 120,
      "Logging Enabled": false,
      "Cache Event Listener Configurations": [],
      "Cache Extension Configurations": [],
      "Bootstrap Cache Loader Factory Configuration": null,
      "Cache Exception Handler Factory Configuration": null,
      "Terracotta Configuration": null,
      "Cache Writer Configuration": {
        "writeMode": "WRITE_THROUGH",
        "notifyListenersOnException": false,
        "minWriteDelay": 1,
        "maxWriteDelay": 1,
        "rateLimitPerSecond": 0,
        "writeCoalescing": false,
        "writeBatching": false,
        "writeBatchSize": 1,
        "retryAttempts": 0,
        "retryAttemptDelaySeconds": 1,
        "cacheWriterFactoryConfiguration": null
      },
      "Cache Loader Configurations": [],
      "Frozen": false,
      "Transactional Mode": "OFF",
      "Statistics": true
    },
    "Cache runtime information": {
      "Statistics enabled": true,
      "Get Time in milliseconds": {
        "Average": 6.3524046E-4,
        "Minimum": 0,
        "Maximum": 0
      },
      "Cache Size": 42144,
      "Accuracy": "Best Effort",
      "Cache Counts": {
        "Hits": 212732032,
        "Miss": 525461,
        "Expired": 0,
        "Eviction": 0,
        "Put": 91710,
        "Remove": 341153
      },
      "Cache Size (Disk)": 0,
      "Cache Count (Disk)": 0
    }
  },
  "OS Information": {
    "OS Name": "Linux",
    "OS Version": "3.10.0-327.36.1.el7.x86_64",
    "System Architecture": "amd64",
    "Available Processors": 8,
    "Average System Load": 18.21
  },
  "Config Git Repository": {
    "Number of commits": 30916,
    "GC Statistics": {
      "sizeOfPackedObjects": 177448255,
      "sizeOfLooseObjects": 0,
      "numberOfPackedObjects": 1105036,
      "numberOfPackFiles": 13,
      "numberOfPackedRefs": 2,
      "numberOfLooseRefs": 1,
      "numberOfLooseObjects": 0
    }
  },
  "Runtime Information": {
    "Name": "38@311b3d3cb61a",
    "Uptime": 172344385,
    "Uptime (in Time Format)": "[About 47 hours, 52 minutes, 24 seconds]",
    "Spec Name": "Java Virtual Machine Specification",
    "Spec Vendor": "Oracle Corporation",
    "Spec Version": "1.7",
    "Input Arguments": [
      "-Djava.security.egd\u003dfile:/dev/./urandom",
      "-Xms512m",
      "-Xmx10g",
      "-XX:PermSize\u003d128m",
      "-XX:MaxPermSize\u003d256m",
      "-Dgo.config.repo.gc.periodic\u003dY",
      "-Duser.timezone\u003dEurope/Stockholm",
      "-Dgo.api.with.safe.mode\u003dfalse",
      "-Dgo.database.provider\u003dcom.thoughtworks.go.postgresql.PostgresqlDatabase",
      "-Duser.language\u003den",
      "-Djruby.rack.request.size.threshold.bytes\u003d30000000",
      "-Duser.country\u003dUS",
      "-Dcruise.config.dir\u003d/etc/go",
      "-Dcruise.config.file\u003d/etc/go/cruise-config.xml",
      "-Dcruise.server.port\u003d8153",
      "-Dcruise.server.ssl.port\u003d8154"
    ],
    "System Properties": {
      "awt.toolkit": "sun.awt.X11.XToolkit",
      "cruise.config.dir": "/etc/go",
      "cruise.config.file": "/etc/go/cruise-config.xml",
      "cruise.server.port": "8153",
      "cruise.server.ssl.port": "8154",
      "file.encoding": "UTF-8",
      "file.encoding.pkg": "sun.io",
      "file.separator": "/",
      "go.api.with.safe.mode": "false",
      "go.config.repo.gc.periodic": "Y",
      "go.database.provider": "com.thoughtworks.go.postgresql.PostgresqlDatabase",
      "java.awt.graphicsenv": "sun.awt.X11GraphicsEnvironment",
      "java.awt.headless": "true",
      "java.awt.printerjob": "sun.print.PSPrinterJob",
      "java.class.path": "/usr/share/go-server/go.jar",
      "java.class.version": "51.0",
      "java.endorsed.dirs": "/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/endorsed",
      "java.ext.dirs": "/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/ext:/usr/java/packages/lib/ext",
      "java.home": "/usr/lib/jvm/java-7-openjdk-amd64/jre",
      "java.io.tmpdir": "/tmp",
      "java.library.path": "/usr/java/packages/lib/amd64:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib",
      "java.net.preferIPv4Stack": "true",
      "java.protocol.handler.pkgs": "com.simontuffs",
      "java.runtime.name": "OpenJDK Runtime Environment",
      "java.runtime.version": "1.7.0_111-b01",
      "java.security.egd": "file:/dev/./urandom",
      "java.specification.name": "Java Platform API Specification",
      "java.specification.vendor": "Oracle Corporation",
      "java.specification.version": "1.7",
      "java.vendor": "Oracle Corporation",
      "java.vendor.url": "http://java.oracle.com/",
      "java.vendor.url.bug": "http://bugreport.sun.com/bugreport/",
      "java.version": "1.7.0_111",
      "java.vm.info": "mixed mode",
      "java.vm.name": "OpenJDK 64-Bit Server VM",
      "java.vm.specification.name": "Java Virtual Machine Specification",
      "java.vm.specification.vendor": "Oracle Corporation",
      "java.vm.specification.version": "1.7",
      "java.vm.vendor": "Oracle Corporation",
      "java.vm.version": "24.111-b01",
      "jetty.base": "/var/lib/go-server",
      "jetty.home": "/var/lib/go-server",
      "jetty.xml.file.name": "jetty.xml",
      "jruby.compile.invokedynamic": "false",
      "jruby.ji.objectProxyCache": "false",
      "jruby.management.enabled": "true",
      "jruby.rack.request.size.threshold.bytes": "30000000",
      "line.separator": "\n",
      "net.sf.ehcache.skipUpdateCheck": "true",
      "os.arch": "amd64",
      "os.name": "Linux",
      "os.version": "3.10.0-327.36.1.el7.x86_64",
      "path.separator": ":",
      "rails.use.compressed.js": "true",
      "sun.arch.data.model": "64",
      "sun.boot.class.path": "/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/resources.jar:/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/rt.jar:/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/sunrsasign.jar:/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/jsse.jar:/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/jce.jar:/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/charsets.jar:/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/rhino.jar:/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/jfr.jar:/usr/lib/jvm/java-7-openjdk-amd64/jre/classes",
      "sun.boot.library.path": "/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64",
      "sun.cpu.endian": "little",
      "sun.cpu.isalist": "",
      "sun.font.fontmanager": "sun.awt.X11FontManager",
      "sun.io.unicode.encoding": "UnicodeLittle",
      "sun.java.command": "/usr/share/go-server/go.jar",
      "sun.java.launcher": "SUN_STANDARD",
      "sun.jnu.encoding": "UTF-8",
      "sun.management.compiler": "HotSpot 64-Bit Tiered Compilers",
      "sun.os.patch.level": "unknown",
      "user.country": "US",
      "user.dir": "/var/lib/go-server",
      "user.home": "/var/go",
      "user.language": "en",
      "user.name": "go",
      "user.timezone": "Europe/Stockholm",
      "webapp.root": "/var/lib/go-server/work/jetty-0.0.0.0-8153-cruise.war-_go-any-/webapp"
    }
  },
  "GC Information": {
    "PS Scavenge": {
      "Memory Pool Names": "[PS Eden Space, PS Survivor Space]",
      "Collection Count": 15909,
      "Collection Time": 983008
    },
    "PS MarkSweep": {
      "Memory Pool Names": "[PS Eden Space, PS Survivor Space, PS Old Gen, PS Perm Gen]",
      "Collection Count": 166,
      "Collection Time": 163262
    }
  },
  "Memory Information": {
    "Heap": {
      "init": 536870912,
      "used": 3294299288,
      "committed": 4062707712,
      "max": 9544663040
    },
    "Non Heap": {
      "init": 136773632,
      "used": 163361376,
      "committed": 183042048,
      "max": 318767104
    },
    "Pending Finalization": 0
  },
  "Memory Pool Information": {
    "Code Cache": {
      "init": 2555904,
      "used": 35261440,
      "committed": 48824320,
      "max": 50331648,
      "Type": "NON_HEAP"
    },
    "PS Eden Space": {
      "init": 135266304,
      "used": 1201615096,
      "committed": 1536163840,
      "max": 3381133312,
      "Type": "HEAP"
    },
    "PS Survivor Space": {
      "init": 22020096,
      "used": 41352584,
      "committed": 100663296,
      "max": 100663296,
      "Type": "HEAP"
    },
    "PS Old Gen": {
      "init": 358088704,
      "used": 2051331608,
      "committed": 2425880576,
      "max": 7158104064,
      "Type": "HEAP"
    },
    "PS Perm Gen": {
      "init": 134217728,
      "used": 128100792,
      "committed": 134217728,
      "max": 268435456,
      "Type": "NON_HEAP"
    }
  },
```

##### Steps to Reproduce

1. Open VSM with "more..." link visible. 
2. Clicking does NOT expand pipeline box. 

##### Expected Results

Should expand box.

##### Actual Results

```
application-b4be538571d908946aceb3f8b2f22229f03ff9d0b384aae69d85287822587bde.js:1846 Uncaught TypeError: Cannot read property 'replace' of null
```
##### Log snippets

Nothing in logs as I could see.

##### Code snippets/Screenshots

![image](https://cloud.githubusercontent.com/assets/1794869/19234906/c8903e4e-8eee-11e6-9f1f-2273170b0568.png)

**Comment by maheshp** [comment]
@chrislovsund this issue is fixed as part of PR https://github.com/gocd/gocd/pull/2786, fix should be available in Go 16.11.0 due soon.

**Comment by rajiesh** [comment]
Verified on GoCD version 16.11.0 (4174-cfb732397132b330e9786d1011c7dc3ba1c9f260), functionality working as expected
